src/JSInterop/Microsoft.JSInterop/src/DotNetObjectRef.cs

```diff
@@ -15,8 +15,7 @@ public static class DotNetObjectRef
         /// <returns>An instance of <see cref="DotNetObjectRef{TValue}" />.</returns>
         public static DotNetObjectRef<TValue> Create<TValue>(TValue value) where TValue : class
         {
-            var objectId = DotNetObjectRefManager.Current.TrackObject(value);
-            return new DotNetObjectRef<TValue>(objectId, value);
+            return new DotNetObjectRef<TValue>(DotNetObjectRefManager.Current, value);
```

Comment:
Is this going to fight with your other change?

Reply:
Yup. But it's easier to review these things piecemeal.